update-submodule/README.md

```diff
@@ -89,6 +99,7 @@ jobs:
   submodule-update:
     name: Submodule update
     runs-on: docker
+    id: submodule-update
```

Comment:
Looks wrong. ID should be inside the step below.

Reply:
fixed